modules/navier_stokes/include/base/NSFVBase.h

```diff
@@ -291,6 +294,8 @@ class NSFVBase : public BaseType
   /// Switch to enable friction correction for the porous medium momentum
   /// equations
   const bool _use_friction_correction;
+  /// Boolean to identify if the residual and jacobian should be computed simultaneoulsy
+  const bool _residual_and_jacobian_together;
```

Comment:
```suggestion
```

Reply:
I just realized I need this since NSVFBase.h doesn't see the parameter unless you suggest inheriting it from elsewhere.

Reply:
I think Alex is thinking to capture it from the Executioner (if set false there), but default to true in NSFV action.

We want to avoid the conflict
NSFVAction setting residual_and_jacobian_together = true (or false)
Executioner (Steady for example) setting residual_and_jacobian_together = false (or true)